### PR TITLE
Adding UI Alerts to the Pattern Library

### PR DESCRIPTION
**Pull request by mfunkie**
Also with this commit is a set of new variables called Shadow Depth.  We can discuss what we want to do with these, and how we might integrate them into existing components but I just wanted to get them in and start using.

### CODE REVIEW

**Comment by mfunkie** [comment]
screen shot 2015-02-24 at 12 38 08 pm

**Comment by mfunkie** [comment]
screen shot 2015-02-24 at 12 38 18 pm

File: less/ui-alert.less [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+.ui-alert__body {
+  margin-left: 50px;
+}
+
+.ui-alert__icon {
+  font-size:   26px;
+  position: absolute;
+  top:  50%;
+  left: 50%;
+  transform: translateY(-50%) translateX(-50%);
```
Comment (mcraiganthony) [inline less/ui-alert.less]: Might wanna shorthand it:
transform: translate(-50%,-50%);

File: less/ui-alert.less [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  }
+}
+
+.ui-alert__icon-wrapper {
+  position:    absolute;
+  top:         0;
+  left:        0;
+  width:       50px;
+  height:      100%;
+  borderColor: transparent;
```
Comment (mcraiganthony) [inline less/ui-alert.less]: prop name should be "border-color"

File: less/ui-alert.less [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+
+.ui-alert {
+  position:         relative;
+  padding:          @padding-half-gutter 24px @padding-half-gutter @padding-half-gutter;
+  background-color: @white;
+  .box-shadow(@shadow-depth-2);
+
+  & + .ui-alert {
+    margin-top: 16px;
```
Comment (mcraiganthony) [inline less/ui-alert.less]: Instead of 16px, use @margin-half-gutter (which is 15px) so that it remains consistent with our page gutters.

**Pull request merged**